### DISCUSSION TITLE
conveyor make copied-site failed

### DISCUSSION THREAD

**Discussion by mzglinicki** [hydraulic-software/conveyor#138] [Q&A]
Hi! 
I am working on KMP desktop app and release it using conveyor. After running conveyor `make copied-site` task failed. Can you help me to solve this problem?

The output:

```
Task failed: Base Configured Inputs for Windows Intel: Input doesn't exist or isn't readable: /Users/.../StudioProjects/driving-time/app/school-app/build/libs/school-app-jvm-1.0.0.jar
Did you mean: school-app-desktop-1.0.0.jar  
```

**Comment by mikehearn** [comment]
You need to run `./gradlew jvmJar` first.

**Reply by mzglinicki** [reply]
Thanks! you are right, the jar file was not generated. 
I was able to integrate my app with conveyor, release with github releases. I am able to install and run the application na mac os but on windows I have following error after starting installation. Do you know why it may happen?
![Przechwytywanie](https://github.com/user-attachments/assets/4edd346a-e953-45e1-96f7-848a764fa6e4)

**Comment by mikehearn** [comment]
That repository appears to be private.

**Reply by mzglinicki** [reply]
yes, for now it is private. I will make it public or use standard pricing plan. What I am expecting for now is problem with certificates. I will check it and back to you.
